requirements/pip.txt

```diff
@@ -1,11 +1,11 @@
 # Base packages
 pip==19.1.1
 appdirs==1.4.3
-virtualenv==16.6.0
+virtualenv==16.6.1
 
 django==1.11.21  # pyup: <1.12
-django-guardian==1.5.1
-django-extensions==2.1.7
+django-guardian==2.0.0
```

Comment:
https://github.com/django-guardian/django-guardian/blob/devel/CHANGES#release-200-june-11-2019

>  Drop support for Python 2.7 & 3.4 and Django 1.11 & 2.0

We can't update this one